### ISSUE TITLE
I have question about data normalize.

### ISSUE THREAD

**Issue by SpicyYeol** [ZitongYu/PhysNet#3]
1. Does the data loader need to normalize the target?
2. Why is the loss value calculated after normalizing the inference result?

**Comment by SpicyYeol** [comment]
1. This is fine because every conv block has a batch norm layer
2. Pearson correlation can be performed on the premise of normalization of input data.

**Comment by mortezagolzan** [comment]
> 1. Does the data loader need to normalize the target?
> 2. Why is the loss value calculated after normalizing the inference result?

Hi @SpicyYeol,
Did you able to train the network?!
